tests/wicked/startandstop/sut/t11_vlan_ifdown_modify_one_config.pm

```diff
@@ -21,12 +21,10 @@ use utils 'file_content_replace';
 
 sub run {
     my ($self, $ctx) = @_;
-    my $config   = '/etc/sysconfig/network/ifcfg-' . $ctx->iface() . '.42';
-    my $local_ip = $self->get_ip(type => 'vlan_changed', netmask => 1);
-    $local_ip =~ s'/'\\/';
+    my $config      = '/etc/sysconfig/network/ifcfg-' . $ctx->iface() . '.42';
     my $previous_ip = $self->get_ip(type => 'vlan', netmask => 1);
     $previous_ip =~ s'/'\\/';
```

Comment:
I think we should escape `/` also in the regex? Should we  do it now or separate PR?
@asmorodskyi WDYT

Reply:
instead of so matching escaping maybe we could just replace `/` with `|` ?

Reply:
see for details https://netjunky.net/sed-replace-path-with-slash-separators/

Reply:
yes we could do it, but then we need to escape `|`

Reply:
follow up: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/7379